samples/subscriptions.js

```diff
@@ -622,6 +603,28 @@ const cli = require(`yargs`)
     },
     opts => listenForMessages(opts.subscriptionName, opts.timeout)
   )
+  .command(
+    `listen-flow-control <subscriptionName>`,
+    `Listen to messages with flow control settings, which are properties of the client/listener instance.`,
+    {
+      maxInProgress: {
+        alias: 'm',
+        type: 'number',
+        default: 0,
+      },
+      timeout: {
+        alias: 't',
+        type: 'number',
+        default: 10,
```

Comment:
`setTimeout` takes milliseconds, is `10` maybe too low?

Reply:
Good call! Updated.

Reply:
I didn't realize we were converting to seconds in the sample, my mistake!